Update Cypress Studio documentation (#4696)
cypress-io/cypress-documentation: content/guides/references/cypress-studio.md

---
title: Cypress Studio

> e2eSpecific: true

Reviewer: Above screenshot seems usually wide?

Also tip - if you hold SPACE during the screenshot, you can get the real clean "capture window only" style like the previous ones - good to keep in mind.

</Alert>

## Limitations

> - Studio is currently only available for writing E2E tests, and doesn't yet work

Reviewer: ```suggestion
- Studio is currently only available for writing End To End tests, and doesn't yet work
```

I think we normally capitalize testing types

>   with component testing.

Reviewer: ```suggestion
  with Component Testing.
```

> - Studio does not support writing tests that use domains of multiple origins.

Reviewer: ```suggestion
- Cypress Studio does not support writing tests that use domains of multiple origins.
```

We tend to refer to features as "Cypress ___"

#### Step 1 - Run the spec

We will use Cypress Studio to perform a "New Transaction" user journey. First,

> launch Cypress and select E2E testing, then choose a browser to run specs in.

Reviewer: ```suggestion
launch Cypress and select End To End testing, then choose a browser to run specs in.
```

Just for consistency (docs are a bit of a mess, gotta start somewhere, heh)
